core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -331,6 +331,15 @@ class ReplicaManager(val config: KafkaConfig,
     delayedFetchPurgatory.checkAndComplete(topicPartitionOperationKey)
   }
 
+  /**
+   * Complete any local follower fetches that have been unblocked since new data is available
+   * from the leader for one or more partitions. Called in ReplicaFetcherThread after successfully
+   * replicating from the leader.
+   */
+  private[server] def completeDelayedFetchRequests(topicPartitions: Seq[TopicPartition]): Unit = {
```

Comment:
maybe rename to : `maybeCompleteDelayedFetchRequests`, since in leader fetch case, the fetch purgatory won't be completed, right?